zorg/jenkins/jobs/jobs/lldb-statistics

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env groovy
+
+// These are the LLVM build directories involved:
```

Comment:
Which reminds me: You should make sure to compile clang-19 with itself (basically the stage2 cmake cache in clang/cmake/caches) if that's not already happening since we want the binary to be as stable as possible.

Reply:
Good point. Updated to a 2-stage build. I'm using the `CLANG_ENABLE_BOOTSTRAP` variable instead of the `Apple-stage2.cmake`, because iiuc the latter would build an optimized build.